components/transfer/TransferConfirmModal.vue

```diff
@@ -0,0 +1,212 @@
+<template>
+  <NeoModal v-model="isModalActive" scroll="clip" @close="emit('close')">
+    <div
+      class="modal-width is-flex is-flex-direction-column is-justify-content-space-between">
+      <div>
+        <div
+          class="is-flex is-justify-content-center is-align-items-center header px-6 py-3">
+          <span class="is-size-5 has-text-weight-bold">
+            {{ $t('teleport.send') }}
+            <span class="is-uppercase">{{ unit }}</span>
+          </span>
+          <NeoIcon
+            class="position-right is-clickable mr-6"
+            icon="close"
+            pack="fas"
+            @click.native="closeModal" />
+        </div>
+        <div
+          class="px-6 is-scrollable"
+          :class="{
+            'is-bordered-top': isExpandList,
+          }">
```

Comment:
```suggestion
          :class="[{
            'is-bordered-top': isExpandList,
          }, 
          'px-6 is-scrollable']">
```

Reply:
I changed, but I'm unsure if it's a better practice.